.github/workflows/unit-tests.yml

```diff
@@ -31,52 +34,128 @@ jobs:
-      - run: echo "PYTHON_VERSION=$(cat .python-version)" >> $GITHUB_ENV
-      - uses: actions/setup-python@v2
+      # Set the value of DJANGO_DB_BACKEND which is used in ci/testsettings.py to
+      # configure django's ORM based on whether we're testing postgres or mysql
+      - name: Set django database backend adapter
+        env:
+          BACKEND: ${{ matrix.db }}
+        run: echo "DJANGO_DB_BACKEND=$(echo "$BACKEND")" >> $GITHUB_ENV
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      # use docker cp to copy the configset, then bash to set ownership to solr
+      - name: Copy Solr configset to solr service
+        run: |
+          docker cp solr_conf ${{ job.services.solr.id }}:/opt/solr/server/solr/configsets/ppa
+          docker exec --user root ${{ job.services.solr.id }} /bin/bash -c "chown -R solr:solr /opt/solr/server/solr/configsets/ppa"
+
+      - name: Copy solr configsets to solr home directory (Solr 8 only)
+        run: "docker exec -d ${{ job.services.solr.id }} cp -r /opt/solr/server/solr/configsets /var/solr/data"
+        if: ${{ matrix.solr == 8.6 }}
+
+      # NOTE: for duration of migration to PUL, use python version from build matrix
+      # Python version to use is stored in the .python-version file, which is the
+      # convention for pyenv: https://github.com/pyenv/pyenv
+      # - name: Get Python version
+      #   run: echo "PYTHON_VERSION=$(cat .python-version)" >> $GITHUB_ENV
+
+      # use python version for current build matrix
+      - name: Setup Python
+        uses: actions/setup-python@v2
         with:
-          python-version: ${{ env.PYTHON_VERSION }}
-      - uses: actions/cache@v2
+          python-version: ${{ matrix.python }}
+
+      - name: Cache pip
+        uses: actions/cache@v2
         with:
           path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          key: pip-${{ matrix.python }}-${{ hashFiles('requirements.txt') }}
           restore-keys: |
-            ${{ runner.os }}-pip-
-      - run: pip install -r dev-requirements.txt
-      - run: cp ci/testsettings.py ppa/local_settings.py
-      - run: python -c "import uuid; print('SECRET_KEY = \'%s\'' % uuid.uuid4())" >> ppa/local_settings.py
-      - run: py.test --cov=./ --cov-report=xml
-      - uses: codecov/codecov-action@v1
+            pip-${{ matrix.python }}-${{ hashFiles('requirements.txt') }}
+            pip-${{ matrix.python }}
+            pip-
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install -r dev-requirements.txt
+
+      # This unnec?
+      # - name: Download webpack stats
+      #   uses: actions/download-artifact@v2
+      #   with:
+      #     name: webpack-stats.json
+
+      # - name: Debug output database configuration
+      #   run: python -c "import pprint; import ci.testsettings; pprint.pprint(ci.testsettings.DATABASES)"
```

Comment:
I can't remember what this was for! I think maybe a shared dependency for  the python tests and js tests... ?

Disabling it didn't cause any problems or warnings?

Reply:
Whew, makes me feel better not remembering it. Sounds like it's fine to assume it doesn't apply here